src/Action/GetShortObjectDescriptionAction.php

```diff
@@ -48,9 +48,9 @@ public function __invoke(Request $request): Response
         $uniqid = $request->get('uniqid');
         $linkParameters = $request->get('linkParameters', []);
 
-        $admin = $this->pool->getInstance($code);
-
-        if (!$admin) {
+        try {
+            $admin = $this->pool->getInstance($code);
```

Comment:
Does the implementation throw an exception?

Reply:
yes, 
the
```php
\Sonata\AdminBundle\Admin\Pool::getInstance(string $id): AdminInterface;
```
throws `\InvalidArgumentException` if admin is not found